Packs/Cybereason/Integrations/Cybereason/Cybereason.py

```diff
@@ -654,10 +752,108 @@ def unisolate_machine(client: Client, machine: str) -> Any:
     return response, pylum_id
 
 
+def malop_connection_command(client: Client, args: dict):
+    malop_guids = args.get('malopGuids')
+    date_time = str(args.get('dateTime'))
+    startTime = args.get('startTime', 0)
+    endTime = args.get('endTime', 9007199254740991)
+
+    filter_input = []
+    if date_time != 'None':
+        date_time_parser = dateparser.parse(date_time)
+        if not date_time_parser:
+            raise DemistoException("dateTime could not be parsed. Please enter a valid time parameter.")
+        date_time_parser = date_time_parser.timestamp()
+        milliseconds = int(date_time_parser * 1000)
+        filter_input = [{"facetName": "creationTime", "filterType": "GreaterThan", "values": [milliseconds], "isResult": True}]
+
+    if isinstance(malop_guids, str):
+        malop_guids = malop_guids.split(',')
+    elif not isinstance(malop_guids, list):
+        raise Exception('malopGuids must be array of strings')
+
+    response = malop_connection(client, malop_guids, filter_input, startTime, endTime)
+    elements = dict_safe_get(response, ['data', 'resultIdToElementDataMap'], default_return_value={}, return_type=dict)
+    outputs = []
+
+    for item in list(elements.values()):
+        simple_values = dict_safe_get(item, ['simpleValues'], default_return_value={}, return_type=dict)
+        element_values = dict_safe_get(item, ['elementValues'], default_return_value={}, return_type=dict)
+        output = {}
+        for info in CONNECTION_INFO:
+            if info.get('type', '') == 'filterData':
```

Comment:
i didnt found inside `CONNECTION_INFO` field with type **filterData**,
did you wanted to check `PROCESS_INFO`?